recipes/libalsa/all/conanfile.py

```diff
@@ -75,3 +75,4 @@ def package_info(self):
         self.cpp_info.names['pkg_config'] = 'alsa'
         self.cpp_info.names["cmake_find_package"] = "ALSA"
         self.cpp_info.names["cmake_find_package_multi"] = "ALSA"
+        self.env_info.ALSA_CONFIG_DIR = os.path.join(self.package_folder, "res")
```

Comment:
Uuuuh. I believe I commited that change a long time ago to a local branch of mine, but I was blocked on a bug in `libtool` and never picked up the work on libalsa after that😅

Reply:
I just found out that [it's been here for a long time actually](https://github.com/conan-community/conan-libalsa/commit/aa4ffc73ee071f771cd0b7289d03e549dac3be5a) , but [we removed it](https://github.com/conan-io/conan-center-index/commit/dac5b7e1c63fd4f391d7cc0ec1aae1400dda31ec#diff-071d5f8987b5a9c403fac167e32c477b22c45b7faa06996341ec9f55d1e78607) because the share folder was not packaged any more (and I just noticed I should add a "alsa" subfolder here)